scripts/l10n/import_native_android_strings.action.sh

```diff
@@ -37,7 +37,7 @@ for TRANSLATION_FILE in "$TRANSLATIONS_DIR/*"; do
   echo "Importing ${LANG}"
   LANG=$(get_native_android_locale $LANG)
 
-  NATIVE_DIR="cordova-plugin-outline/android/resources/strings/values-$LANG"
+  NATIVE_DIR="src/cordova/plugin/android/resources/strings/values-$LANG"
```

Comment:
Do we need to be running these?

Reply:
Probably? I never really touched this code. My guess is that it extracts the Android messages from our messages.

Reply:
Perhaps we haven't changed the messages we use in the Android native code. You'd have to audit where it's being used.

Reply:
@jyyi1 in case they know

Reply:
I am not aware of it either, probably it is used by some log messages sent from the native code, which is not directly visible to the user.

Reply:
Okay, I'll address this in the next pr where i break up the root `scripts` folder